packages/graphiql/src/components/DocExplorer.tsx

```diff
@@ -73,7 +74,7 @@ export class DocExplorer extends React.Component<
   }
 
   render() {
-    const { schema } = this.props;
+    const { schema, t } = this.props;
```

Comment:
this component is being moved to a new package, however I *think* these changes will follow without much conflict when we merge all the moved indexes